ledger/Cargo.toml

```diff
@@ -60,7 +60,8 @@ trees = "0.4.2"
 [dependencies.rocksdb]
 # Avoid the vendored bzip2 within rocksdb-sys that can cause linker conflicts
 # when also using the bzip2 crate
-version = "0.18.0"
+git = "https://github.com/rust-rocksdb/rust-rocksdb"
+rev = "ac4ffa3ebe5b92bff3936deac36edabc4c3f24a5"
```

Comment:
I didn't see anywhere else where we do this in the repo, and it is unclear to me if we'd be ok shipping against a non-tagged commit. Maybe ask about this in Discord development channel as others may have opinions one way or another?

I get that the obvious drawback of using tagged commit only is that we have to wait for maintainers of rust-rocksdb to push another release

Reply:
> Yep, that's the main downside, and their last release is already a year ago :(.

Technically they've had a [more recent release](https://github.com/rust-rocksdb/rust-rocksdb/releases/tag/v0.18.0
), but your point is valid either way about us wanting to be able to consume RocksDB changes quicker. If we do want to stick to releases, maybe the rust-rocksdb folks would be amenable to us helping with release process / overhead.